### PR TITLE
adds popecrunch cereal radio ad

### PR DESCRIPTION
**Pull request by Adharainspace**
## About the PR 

voices: Puppet Master and HeadsmanStukka of the Black Pants Legion

backing audio: "Smooth Talker" by Apoxode 2019 - Licensed under Creative Commons Attribution (3.0)

editor: Phuzzy

(i added it to the ship one and the debris map)

## Why's this needed? 
more ad variety!!

## Changelog

```
(u)Phuzzy:
(*)Adds a new ad! Find it at your local space radio station!
```

### CODE REVIEW

**Comment by PhuzzySA** [comment]
Thanks again to the fine folks in #imcoder for getting this put together. I also want to thank the guys at BPL for humoring me with this, and hope to continue that relationship.

**Review by Sovexe — CHANGES_REQUESTED** [review]
image

**Comment by Adharainspace** [comment]
image
fixed!

**Comment by Adharainspace** [comment]
fixed it!

File: code/modules/telescience/radiostation.dm [hunk 1]
```diff
@@ -613,6 +613,13 @@
 	audio = "sound/radio_station/bill_movie.ogg"
 	name_of_thing = "some shitty movie"
 
+/obj/item/radio_tape/advertisement/pope_crunch
+	name = "compact tape - Pope Crunch"
+	audio = "sound/radio_station/pope_crunch_cereal.ogg"
+	name_of_thing = "Pope Crunch Cereal"
+	desc = {"A small audio tape. Though, it looks to big to fit in an audio log.<br>
+	The music is Smooth Talker by Apoxode licensed under Creative Commons Attribution (3.0)"}
```
Comment (Sovexe) [inline code/modules/telescience/radiostation.dm:621]:
```suggestion
/obj/item/radio_tape/advertisement/pope_crunch
	name = "compact tape - 'Pope Crunch'"
	audio = "sound/radio_station/pope_crunch_cereal.ogg"
	name_of_thing = "Pope Crunch Cereal"
	desc = {"A small audio tape. Though, it looks to big to fit in an audio log.<br>
	The music is Smooth Talker by Apoxode (CC BY 3.0)"}
```
consistency with other tapes

Comment (Adharainspace) [reply code/modules/telescience/radiostation.dm]: got it thanks

**Review by Sovexe — APPROVED** [review]

**Comment by Adharainspace** [comment]
ok now ive really fixed it

**Pull request merged**